daemon/integration/integration_test.go

```diff
@@ -0,0 +1,66 @@
+package integration_test
+
+import (
+	"fmt"
+	"github.com/scootdev/scoot/daemon/client/cli"
+	"github.com/scootdev/scoot/daemon/client/conn"
+	"github.com/scootdev/scoot/daemon/integration"
+	"github.com/scootdev/scoot/daemon/server"
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+)
+
+func TestIntegration(t *testing.T) {
+	tempDir, err := ioutil.TempDir("", "scoot-listen-")
+	if err != nil {
+		t.Fatal("could not make temp directory", err)
+	}
+	// defer os.RemoveAll(tempDir)
```

Comment:
It is not necessary (the tmpreaper will eventually get it), but good to do. I accidentally pushed code as I was testing it; whoops!

Fixed.